README.md

```diff
@@ -41,7 +41,7 @@ version: 2.1
 # If you don't have a top-level `orbs` section, add one
 orbs:
 # Add the Rainforest orb to that list
-  - rainforest: rainforest-qa/rainforest@3.0.1
+  - rainforest: rainforest-qa/rainforest@3
```

Comment:
I think this should be 3.0.2?

Reply:
or does this mean latest v3.x.x?

Reply:
I think it should be unspecific, otherwise everyone has to update it themselves each release...I took the example from https://github.com/rainforestapp/regenwald/blob/develop/.circleci/config.yml#L50